src/Runner.Worker/ActionManager.cs

```diff
@@ -400,6 +400,8 @@ public Definition LoadAction(IExecutionContext executionContext, Pipelines.Actio
                         var compositeAction = definition.Data.Execution as CompositeActionExecutionData;
                         Trace.Info($"Load {compositeAction.Steps.Count} action steps.");
                         Trace.Verbose($"Details: {StringUtil.ConvertToJson(compositeAction.Steps)}");
+                        Trace.Info($"Load: {compositeAction.Outputs} outputs");
```

Comment:
does this print anything useful, or does it print "object"

Reply:
`compositeAction.Outputs.Count`

Reply:
Good catch - thanks!

Reply:
beware null ref

Reply:
`compositeAction.Outputs?.Count ?? 0`